src/test/java/com/okta/tools/aws/settings/ConfigurationTest.java

```diff
@@ -238,46 +243,4 @@ void addOrUpdateProfileToExistingProfile() throws IOException {
 
         assertEquals(expected, given);
     }
-
-    /*
-     * Tests whether the Reader given to the Configuration constructor is properly closed.
-     */
-    @Test
-    public void constructorClosesReader() throws Exception {
-        final String simpleIniDocument = "[ini]\nfoo=bar";
-        final StringReader reader = new StringReader(simpleIniDocument);
-
-        // This should consume reader
-        new Configuration(reader);
-        // Causing this to throw an exception
-        assertThrows(IOException.class, () -> reader.ready(), "Stream closed");
-    }
```

Comment:
This was an annoying behavior of Ini4j. Thankfully it is not preserved in configuration2.